src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -1146,15 +1147,25 @@ MsalIdTokenCacheItem ITokenCacheInternal.GetIdTokenCacheItem(MsalAccessTokenCach
                 tenantProfiles?.Values);
         }
 
+        async Task ITokenCacheInternal.RemoveLongRunningOboAccountAsync(string longRunningOboCacheKey, AuthenticationRequestParameters requestParameters)
+        {
+            await RemoveAccountAsync(null, longRunningOboCacheKey, requestParameters).ConfigureAwait(false);
```

Comment:
It doesn't feel right to reuse `RemoveAccount` like this. I think you need new api.